TUM Campus App/CardTableViewCell.swift

```diff
@@ -7,12 +7,41 @@
 //
 
 import UIKit
+import FoldingCell
 import MCSwipeTableViewCell
 
-class CardTableViewCell: MCSwipeTableViewCell {
+class CardTableViewCell: FoldingCell {
+    
     
     func setElement(_ element: DataElement) {
```

Comment:
MCSwipeTableViewCell is there for the swipe gestures. 
Depends on whether we don't want those anymore.

You decide. 'Cause I'm a terrible designer